bundles/core/org.eclipse.smarthome.core.transform/OSGI-INF/transformationaction.xml

```diff
@@ -12,6 +12,6 @@
 	<implementation class="org.eclipse.smarthome.core.transform.internal.TransformationActionService" />
 
 	<service>
-		<provide interface="org.eclipse.smarthome.core.scriptengine.action.ActionService" />
+		<provide interface="org.eclipse.smarthome.model.script.actions.ActionService" />
```

Comment:
Isn't the interface "org.eclipse.smarthome.model.script.engine.action.ActionService" now?